policies/travel-policy.md

```diff
@@ -0,0 +1,352 @@
+
+Only the OMC can appoint Travel Approvers and amend the ***Travel
+Approver Table***.
+
+Travel Approver's can only authorise travel up to the limits specified
+in the ***Travel Approver Table***.
+
+  
+  | Travel Approver Table | Total Value of Travel |
+  | --- | --- |
+  | Manager and OMC member | US\$15,000 |
+  | Travel over US\$15,000 can only get approval direct from the OMC |  
+  
+
+### Travel Arrangements
+
+The Traveller is responsible for making and paying for all travel
+arrangements including but not limited to; booking and payment of all
+transportation, accommodation, conference fees, meals and travel-related
+expenses.
+
+### International Travel
+
+-   International business trips should be authorized at least 2 months
+    before the expected travel date, where practical.
+-   Visas or other fees related to border requirements are eligible for
+    reimbursement.
+-   Travel Insurance is needed for all international travel and is
+    reimbursable. Where the travel insurance covers a period of travel
+    unrelated to business (e.g. where you add on extra days) the cost of
+    the insurance should be on a pro-rata basis as only the business
+    portion is reimbursable.
+-   Any medical expenses not covered under the Travel Insurance, are at
+    the Traveller's own cost.
+
+### Transportation
+
+The Traveller can claim reimbursement for any means of transportation
+that is needed to fulfil their duties to OpenSSL. Including a return
+plane, boat, train, coach ticket and/or rental car subject to the limits
+and restrictions listed in The Policy.
+
+The Traveller should travel using the most effective and economical
+means, taking into account the distance and time taken to travel.
+
+#### Taxis
+
+While traveling, Travellers will be reimbursed when using a taxi service
+to and from; their home, the airport, hotel, and business location, all
+other taxi fares are at the Traveller's expense.
+
+#### Flights
+
+Flights should be booked as per the ***Flight Class Table*** below and
+are inclusive of 1 checked piece of luggage per person.
+
+-   If the Traveller chooses to upgrade their flight the difference
+    between the cost of the approved flight class and the upgraded
+    flight class is at the Traveller's own expense.
+-   Additional baggage and/or overweight baggage fees are at the
+    Traveller's own cost.
+- 	Airport parking is also possible utilize
+
+##### Flight Class Table
+ Total Flying Time is the time from the scheduled departure of the first flight to the scheduled arrival of the last flight
+ 
+ | Total Flying Time | Flight Class | Where Flight Class is not available |
+ | --- | --- | ---|
+ | Less than 6 hours | Economy| n/a |
+ | Greater than 6 but less than 12 hours | Premium Economy | Economy with an extra pre-arrival day (i.e. arrives 2 days prior) |
+ | Greater than 12 hours | Business Class | Premium Economy or Economy with an extra pre-arrival day (i.e. arrives 2 days prior) |
+
```

Comment:
To arrive 2 days prior and lose the comfort of traveling doesn't make sense. These are 2 different things. Suppose I come 2 days before I get a benefit to settle myself from jet lag. It doesn't help if I fly for more than 6 hours. 
So I would omit this column "Where flight class not available" .